framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -2327,4 +2232,37 @@ public AnnotatedTypeMirror getDummyAssignedTo(ExpressionTree expressionTree) {
         }
         return null;
     }
+
+    /**
+     * Evaluates a given annotation to its primitive integral TypeKind equivalent.
+     *
+     * @param type {@link javax.lang.model.type.TypeKind}
+     * @return one of INT, SHORT, BYTE, CHAR, LONG
+     */
+    private static final TypeKind toPrimitiveIntegralTypeKind(TypeMirror type) {
```

Comment:
The `final` doesn't add anything for a `private` method.

Reply:
You are right.  I will see that it is removed.